src/interactive/Popover.jsx

```diff
@@ -160,10 +160,12 @@ class Popover extends React.Component {
 				'popover-container',
 				'popover-container--menu',
 				{
-					'trans-fadeIn--short': !isActive,
+					'trans-fadeIn': !isActive,
```

Comment:
Stupid nitpick...there's a lot of classnames here, so writing them like this might help it look less overwhelming:
```
'trans-fadeIn opacity--0 visibility--a11yHide': !isActive,
'trans-fadeOut opacity--1 visibility--a11yShow': isActive,
'popover-container--horizontal-left': (align === 'left'),
'popover-container--horizontal-right': (align === 'right')
```